language/ir-testsuite/tests/validator_set/reconfiguration_via_validator_addition.mvir

```diff
@@ -22,9 +22,14 @@ main() {
     return;
 }
 
+//! block-prologue
+//! proposer: vivian
+//! block-time: 2
+
 //! new-transaction
 //! sender: association
-// run a tx from the association that adds Alice and Bob as validators
+// run a tx from the association that adds Alice and Bob as validators, this will fail as only one reconfiguration is
```

Comment:
I don't understand--the tx below only adds one validator.

Reply:
It adds two validators right?
`LibraSystem.add_validator({{alice}});
  LibraSystem.add_validator({{bob}});`
The code display doesn't look quite right as a few lines are skipped.